src/proteomes/components/landing-page/styles/landing-page.module.scss

```diff
@@ -0,0 +1,68 @@
+@import 'franklin-sites/src/styles/settings';
+@import 'franklin-sites/src/styles/colours';
```

Comment:
Following replacing SASS variables with CSS Custom Properties, we can remove this import as the colours are set globally

Reply:
I am still using $global-margin from settings. Shall I keep it as it is or set the size?

Reply:
For now we don't have Custom Properties for sizing (yet), so you can only remove `colours`, but you'll need to keep `settings` because of that global margin variable